SessionState/__test__/handler.test.ts

```diff
@@ -132,5 +132,30 @@ describe("SessionState", () => {
       kind: "IResponseErrorGatewayTimeout"
     });
   });
+
+  test(`GIVEN a valid user decoded from JWT
+    WHEN the client returns an error
+      THEN the response should be 500`, async () => {
```

Comment:
As a suggestion, in future consider the option of making similar tests parametric using `it.each`.